docker-compose.yml

```diff
@@ -13,3 +13,30 @@ services:
       - /app/.next
     ports:
       - '3000:3000'
+    environment:
+      - MONGO_URL=mongodb://mongo:27017
+      - MONGODB_DB=dev
```

Comment:
super tiny nit: `MONGO_DB_NAME` might be a little clearer that this is the name of the database?